### PR TITLE
made styling and quality of life fixes to reserve and edit map pages

### PR DESCRIPTION
**Pull request by willmt80**

### CODE REVIEW

**Comment by willmt80** [comment]
new styling

File: src/views/MapView.vue [hunk 1]
```diff
@@ -25,22 +25,15 @@
...
-    <div class="header-bar">
-      <!-- <b-button v-if="reservedFilter === 0" disabled>Available blocks</b-button> -->
-      <!-- <b-button v-if="reservedFilter === 0" disabled>Blocks near me</b-button> -->
-      <h3 v-if="reservedFilter === 1 && !!activeStreetId">Block {{this.activeStreetId}}</h3>
-      <b-button v-on:click="labelsVisible = !labelsVisible">
-        {{`${(this.labelsVisible ? "Hide" : "Show")} block labels`}}
-      </b-button>
-    </div>
+
+    <hr />
```
Comment (jackblanc) [inline src/views/MapView.vue:29]: what is this? can we give it a more descriptive name?

Comment (willmt80) [reply src/views/MapView.vue:29]: It's a horizontal rule, it's a built in element. It's that line between the header and the map

**Comment by willmt80** [comment]
bigger map

File: src/components/Map.vue [hunk 2]
```diff
@@ -26,40 +30,46 @@ export default {
...
-      // TODO: find way to perform function on ESRI data;
-        return '<b>ID:</b> {ID} <strong>RESERVED:</strong> {RESERVED}';
+      function getModalContent(feature) {
+        let reserveString = 'Open';
+        if (feature.graphic.attributes.RESERVED === '1') {
+          reserveString = 'Reserved';
+        } else if (feature.graphic.attributes.RESERVED === '2') {
+          reserveString = 'Complete';
+        }
+        // const reserved = '{RESERVED}' === '0' ? 'Open' : 'Reserved';
```
Comment (liammoyn) [inline src/components/Map.vue:70]: This TODO is obsolete right?

Comment (willmt80) [reply src/components/Map.vue:70]: Yes that was from a couple months ago

**Pull request merged**